cucumber/kubernetes/features/login_errors.feature

```diff
@@ -15,12 +15,12 @@ Feature: Errors emitted by the login method.
   Scenario: it raises an error when logging in as a host which does not hold "authenticate" privilege
     on the webservice.
     When I login to authn-k8s as "pod/inventory-unauthorized"
-    Then the HTTP status is "401"
+    Then the HTTP status is "403"
 
   Scenario: it raises an error when logging in with a custom prefix as a host which does not
     hold "authenticate" privilege on the webservice.
     When I login to authn-k8s as "@namespace@/pod/inventory-unauthorized" with prefix "host/conjur/authn-k8s/minikube/apps"
-    Then the HTTP status is "401"
+    Then the HTTP status is "403"
```

Comment:
how did our tests pass previously if the error status wasn't correct?

Reply:
This PR actually changes the returned error to a better one. 403 Forbidden is a proper error in this use-case.